src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java

```diff
@@ -152,6 +156,10 @@ public MesosSlaveInfo(
     this.additionalURIs = additionalURIs;
     this.nodeProperties.replaceBy(nodeProperties == null ? new ArrayList<NodeProperty<?>>() : nodeProperties);
 
+    // Ensure minExecutors is at least equal to 1 and prevent case where minExecutors > maxExecutors.
+    int minExecutorsVal = Integer.parseInt(minExecutors);
+    this.minExecutors = minExecutorsVal < 1 || minExecutorsVal > this.maxExecutors ? 1 : minExecutorsVal;
```

Comment:
I think min > max is an anormal case, most likely the result of a typing error when configuring a slave. In my opinion the safest fallback is to set the value to 1 so the admin might notice the wrong number of executors but without provisioning the unnecessary resources that the maxExecutors value would lead to.